server/helpers/funtions.js

```diff
@@ -1,4 +1,7 @@
-const getUserName = (user) =>
+export const getUserName = (user) =>
   user.vendor.companyName ? user.vendor.companyName : `${user.firstName} ${user.lastName}`;
 
-export default getUserName;
+export const getMoneyFormat = (number) =>
+  new Intl.NumberFormat('en-NG', { style: 'currency', currency: 'NGN' }).format(number);
+
+export const getFormattedPropertyName = (property) => `"${property.name}"`;
```

Comment:
Lets make it `getFormattedName` so that we can use it for other names too if necessary.

```suggestion
export const getFormattedName = (name) => `"${name}"`;
```